python/cucim/MANIFEST.in

```diff
@@ -1,24 +1,13 @@
 # https://packaging.python.org/guides/using-manifest-in/
-graft docs
-graft src
-graft ci
-graft tests
-
-include .bumpversion.cfg
-include .coveragerc
-include .cookiecutterrc
 include .editorconfig
 
-include VERSION
-include AUTHORS.md
+include src/cucim/VERSION
 include CHANGELOG.md
 include CONTRIBUTING.md
```

Comment:
It seems that CONTRIBUTING.md is removed in this PR but this statement still exists.

Reply:
```suggestion
```

Reply:
This is now a symlink